relops/cqlc
# Support set map value by key and fix dependency

This PR fixes #36 and #39 Since the repo has not been maintained for a while I changed the readme and test, if it's possible to get those things merged, I can create a cleaner PR with different readme.

Improvements
 
- use latest gocql and fix constant naming #36 
- support set map value by key, previously only entire map is supported #39
- use docker for e2e test
- clean up the template so it won't have too many blank lines and block proper gofmt

Dependencies

- remove seelog, it is only used in generator cli, use standard log is enough
- use `pkg/errrors` to preserve context in cqlc runtime
- use logrus in cqlc runtime, could add a extra logger interface to allow using other library if really needed

Know issues

- integration test no longer works because they generate file in /tmp and can't use gocql under /vendor and will fail assertion due to type mismatch

And @0x6e6562 if you no longer have the time to maintain it, is it possible to update the repo doc and point to a maintained fork, thx.
